manifests/repo/debian.pp

```diff
@@ -62,4 +62,35 @@
       }
     }
   }
+
+  if ($php::globals::php_version == '7.1') {
+    # Required packages for PHP 7.1 repository
+    package { 'apt-transport-https':
```

Comment:
I see conflict potential as well in the packages. So I'm also not sure.

If adding repos is not part of this module, then what's the purpose of this file `debian.pp`?